horace_core/sqw/@DnDBase/sqw_eval_nopix.m

```diff
@@ -0,0 +1,8 @@
+function  wout = sqw_eval_nopix(wout, sqwfunc, all_bins, pars)
+
+% Helper function for sqw eval executed on a pixel-less object (DnD with no pixels
```

Comment:
sqw/dnd?

Reply:
"dnd or sqw with no pixels" have not looked at this function carefully. Man be should be a bit generalized.